### PR TITLE
Fix annotsv TCL error

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-structural/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by nvnieuwk** [comment]
Not going to bother fixing the gridss test now since it will be temporarily deprecated in the next PR

File: .prettierignore
Comment (mvheetve) [inline .prettierignore:1]: what does this file do?

**Review by mvheetve — APPROVED** [review]
Indeed a very small one.

File: .prettierignore
Comment (nvnieuwk) [reply .prettierignore:1]: It tells the `prettier` linting which files to ignore

**Pull request merged**